data/helpers.d/setting

```diff
@@ -274,7 +275,18 @@ ynh_permission_create() {
         allowed=",allowed=['${allowed//';'/"','"}']"
     fi
 
-    yunohost tools shell -c "from yunohost.permission import permission_create; permission_create('$app.$permission', url=$url ${allowed:-} , sync_perm=False)"
+    # Check if permission already exists
+    if ynh_permission_exists --permission $permission; then
+        # If permission exits, update it
+        local add
+        if [[ -n ${add:-} ]]; then
+            add="--add ${allowed//';'/" "}"
```

Comment:
Are you sure of that. The `allowed` variable already contains the good value

Reply:
If `allowed` is empty, then the helper is called like that: `ynh_permission_update --permission $permission --add` and getops don't like it.

But the condition above is wrong

Reply:
Well, I don't know if it's really a good idea to support the case of a permission already exist for this helper. Because if we support this we need to support all situation. Like when we allow a new user but one user is already allowed. Or when we allow nobody but somebody is already allowed...

Reply:
I think we could just think that it's the work of the packager to check that the permission exist. And on the core side we just return an error when we try to create a permission which already exist.

Reply:
You probably right, adding the function `ynh_permission_exists` is enough